content/en/blog/2023/announcing-notation-rc2.md

```diff
@@ -0,0 +1,96 @@
+sample.registry.io/ratify-sample-repo@sha256:5d7a0742f9c17400d21b29d2f27ed1b3429f0a71c5f53fb2a9ca3eff7850d2a6
+└── application/vnd.cncf.notary.signature
+    └── sha256:d9d98d2b56b77f56ebe8e917643c6484017a39b89ff65e8b3449598d6b1adda5
+        ├── media type: application/cose
+        ├── signature algorithm: RSASSA-PSS-SHA-256
+        ├── signed attributes
+        │   ├── signingScheme: notary.x509
+        │   ├── signingTime: Sun Feb 19 15:29:47 2023
+        │   └── expiry: Mon Jan  1 00:00:00 0001
+        ├── user defined attributes
+        │   └── io.wabbit-networks.buildId: 123
+        ├── unsigned attributes
+        │   └── signingAgent: Notation/1.0.0 kms/v0.4.0-beta.1
+        ├── certificates
+        │   └── SHA1 fingerprint: def4344f733f1f57af3efd759b47c4576a10a723
+        │       ├── issued to: CN=wabbit-networks.io,O=Notary,L=Seattle,ST=WA,C=US
+        │       ├── issued by: CN=wabbit-networks.io,O=Notary,L=Seattle,ST=WA,C=US
+        │       └── expiry: Mon Feb 12 16:13:48 2024
+        └── signed artifact
+            ├── media type: application/vnd.docker.distribution.manifest.v2+json
+            ├── digest: sha256:5d7a0742f9c17400d21b29d2f27ed1b3429f0a71c5f53fb2a9ca3eff7850d2a6
+            └── size: 942
+```
+
+When verifying signatures associated with a signed artifact, users may want to get a list of the signed metadata included with the signature and use it evaluate additional decisions before using the signed image. A new flag `--user-metadata` was introduced to `notation sign` and `notation verify` in this release. Similar to annotations, you can easily add user-defined metadata to signature payload when signing an artifact or verify that provided key-value pairs are present in the payload of the valid signature. 
+
+```
+$ notation verify --user-metadata io.wabbit-networks.buildId=123 sample.registry.io:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+Successfully verified signature for sample.registry.io:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+The artifact is signed with the following user metadata.
+
+KEY                         VALUE
+io.wabbit-networks.buildId  123
+```
+
+In addition, storing signatures in the registry using OCI image manifest was added into RC.3, you can use a flag `--signature-format image` to explicitly specify uploading image manifest to a registry when signing an artifact.
+
+Last but not least, `--debug` and `--verbose` flags were added to all CLI commands, providing debug and troubleshooting capability for Notation CLI.
+
+### Ecosystem integration
+
+[Ratify](https://github.com/deislabs/ratify) is a verification engine for Kubernetes which enables verification of artifact security metadata and admits for deployment only those that comply with policies you create. It integrated Notation-go v1.0.0-rc.2 which allows you to verify a container image signed by Notation.
```

Comment:
Do we need to say which release of Ratify integrated Notation RC-2 ?

Reply:
Added the detailed version of Ratify